src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -130,11 +134,18 @@ public static bool ReportAndCatchUnlessCanceled(Exception exception)
         }
 
         /// <summary>
-        /// Use in an exception filter to report a non-fatal error. 
-        /// Calls <see cref="NonFatalHandler"/> unless the operation has been cancelled. 
-        /// The exception isn't passed through (the method returns true).
+        /// Use in an exception filter to report a non-fatal error (by calling <see cref="NonFatalHandler"/>) and catch
+        /// the exception, unless the operation was cancelled at the request of a caller.
```

Comment:
>at the request of a caller. [](start = 62, length = 27)

I think the summary should clearly explain how the method detects that.

Reply:
Quick Info shows both `<summary>` and `<remarks>`. I was hoping this part was clear:
![image](https://user-images.githubusercontent.com/1408396/113021829-d3a60100-9138-11eb-8e72-80457b30d206.png)

Reply:
I personally don't see any benefit in splitting this info between different sections of the doc comment. Is there a reason?

---
In reply to: [604244461](https://github.com/dotnet/roslyn/pull/52253#discussion_r604244461) [](ancestors = 604244461)

Reply:
In my opinion the section that introduces a term "cancelled at the request of a caller" should provide an explanation of the term.

---
In reply to: [604287824](https://github.com/dotnet/roslyn/pull/52253#discussion_r604287824) [](ancestors = 604287824,604244461)